lua/xcodebuild/integrations/xcode_build_server.lua

```diff
@@ -0,0 +1,17 @@
+---@mod xcodebuild.integrations.xcode_build_server xcode-build-server Integration
+---@brief [[
+---This module is responsible for the integration with xcode-build-server.
+---It provides a function to update buildServer.json file with the current scheme.
+---@brief ]]
+
+local M = {}
+
+---Calls "config" command of xcode-build-server in order to update buildServer.json file.
+---@param projectCommand string # either "-project 'path/to/project.xcodeproj'" or "-workspace 'path/to/workspace.xcworkspace'"
```

Comment:
For params you don't need `#` as far as I remember

Reply:
Yep, you're right, I've noticed that it included # in the generated file, but it felt a little weird not to put it 😅